src/charm.py

```diff
@@ -26,6 +21,9 @@ class MySQLOperatorCharm(CharmBase):
     def __init__(self, *args):
         super().__init__(*args)
 
+        # Please do not reference this variable directly. Instead use _get_mysql_helpers().
+        self._mysql_helpers = None
+
```

Comment:
Perhaps I'm slow, what's the purpose of this line if we have the property below?

If we removed the `if` from the `_mysql()` property, always returning a new instance of `MySQL` what would be the impact?

Reply:
The `_mysql()` property will in a future PR retrieve values from the peer relation databag to instantiate a new instance of `MySQL` (inputs to the constructor are currently stubbed). The idea behind caching in the object is to avoid retrieving from the databag over again. However, now that you point this out, may be it's a good idea not to cache and to instead re-instantiate (I dont imagine that retrieving from the peer databag is too expensive)